README.md

```diff
@@ -112,23 +153,28 @@ type Event struct {
 	Field2 string `json:"field2"`
 }
 
-func eventHandlerFunc(msgPayload[]byte, msgHeaders map[string]string, inputs map[string]string) ([]byte, map[string]string, error){
+func eventHandlerFunc(msgPayload any, msgHeaders map[string]string, inputs map[string]string) ([]byte, map[string]string, error){
+    // Type assert to user type. By default this is a *[]byte
+    as_bytes, ok := msgPayload.(*[]byte)
```

Comment:
change *[]byte to []byte

Reply:
https://github.com/memphisdev/memphis-functions.go/pull/6/commits/19bc8ecb18e828936b92ad5c0db770af4fb0bf8d

Done